README.md

```diff
@@ -88,47 +88,56 @@ Supported versions:
     apt-get install aztfexport
     ```
 
-## Precondition
+#### AUR (Linux)
 
-`aztfexport` requires a `terraform` executable installed in the `$PATH`, whose version `>= v0.12`.
+```bash
+yay -S aztfexport
+```
 
-## Usage
+## Prerequisites
 
-Follow the [authentication guide](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs#authenticating-to-azure) from the Terraform AzureRM provider to authenticate to Azure.
+`aztfexport` requires a `terraform` executable installed in the `$PATH` with version `>= v0.12`.
 
-Then you can go ahead and run `aztfexport resource <resource id>`, `aztfexport resource-group <resource group name>` or `aztfexport query <arg where predicate>` to import a single resource, a resource group including child resources, or a customized set of resources by an Azure Resource Graph query.
+## Usage
 
-### Export a Single Resource
+Follow this [authentication guide](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs#authenticating-to-azure) from the Terraform AzureRM provider to authenticate to Azure.
 
-`aztfexport resource [option] <resource id>` exports a single resource by its Azure control plane ID.
+Then run Azure Export for Terraform in the following format:
+```terminal
+aztfexport [command] [option] <scope>
+```
+Azure Export supports three variations of this core usage covered in these next sectionss: 
+- `aztfexport resource <resource id>` to import a single resource
+- `aztfexport resource-group <resource group name>` to import a resource group including child resources
+- `aztfexport query <arg where predicate>` to import a customized set of resources by an Azure Resource Graph query
```

Comment:
Can we simplify this a bit as the command lines are all duplicated in their own section?